spec/swagger.yaml

```diff
@@ -2249,1383 +2405,1328 @@ definitions:
+            ],
+            "effective_from": "2017-07-11T20:31:25.000Z",
+            "effective_to": "2017-07-11T20:32:25.000Z",
+            "_thread": "123"
           },
           "links": {
-            "twine_web_app": "http://10.0.1.55:9000/#/coach/patient/5817ead56f5316c0d05ba8bf",
-            "self": "/pub/patient/5817ead56f5316c0d05ba8bf"
+            "self": "/bundle/596535d9608f63d531b7e34a"
           },
           "relationships": {
-            "groups": {
-              "data": [{
-                "type": "group",
-                "id": "5755db2a3db4179179999acf"
-              }],
+            "plan": {
               "links": {
-                "related": "/pub/patient/5817ead56f5316c0d05ba8bf/groups"
+                "related": "/bundle/596535d9608f63d531b7e34a/plan"
               }
             },
-            "coaches": {
-              "data": [],
+            "actions": {
               "links": {
-                "related":"/pub/patient/5817ead56f5316c0d05ba8bf/coaches"
-              }
+                "related": "/bundle/596535d9608f63d531b7e34a/actions"
+              },
+              "data": [
+                {
+                  "type": "action",
+                  "id": "596535d9608f63d531b7e344"
+                }
+              ]
             }
           }
         },
-        "included": []
+        "included": [],
+        "meta": {
+          "req_id": "c0f8d53a-d763-4beb-b0b1-9ec6da146a14"
+        }
       }
-   
-  # END /patient definitions        
-  
-  # BEGIN /coach definitions
-  CoachResource:
-      type: object
-      required:
-        - id
-        - type
-        - attributes
-      properties:
-        id:
-          type: string
-          example: '57b36e3c04ad8c2224f2af38'
-        type:
-          type: string
-          enum:
-            - coach
-          example: coach
-        attributes:
-          type: object
-          properties:
-            first_name:
-              type: string
-              example: Clarisa
-            last_name:
-              type: string
-              example: Rivera
-        links:
-          type: object
-          readOnly: true
-          required:
-            - self
-          properties:
-            self:
-              type: string
-              pattern: '/coach/[0-9a-z]+'
-              example: '/coach/57b36e3c04ad8c2224f2af38'
-  FetchCoachResponse:
+  # END /bundle definitions
+
+  # BEGIN /calendar_event definitions
+  CalendarEventResource:
+     type: object
+     required:
+       - type
+     properties:
+       type:
+         type: string
+       id:
+         type: string
+       attributes:
+         type: object
+         properties:
+          _completed_by:
+            type: object
+            description: 'The coach who marked the calendar event as completed'
+          completed_at:
+            type: string
+            format: ISODate
+            example: '2017-11-03T06:00:00.000Z'
```

Comment:
This is an unrealistic value for `completed_at`. A typical value will have minutes, seconds, and milliseconds.